DemoUtil/JonSkeet.DemoUtil/targets/JonSkeet.DemoUtil.targets

```diff
@@ -1,17 +1,13 @@
 <Project>
-  <PropertyGroup>
-    <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
-  </PropertyGroup>
-
-  <PropertyGroup Condition="'$(DisableAutoGeneratedMain)' != 'true'">
+  <PropertyGroup Condition="'$(AutoGenerateEntryPoint)' == 'true'">
```

Comment:
This is changing the default to *not* autogenerating the entry point though, isn't it? I very definitely want the default to be "you add the package reference and everything just happens automatically."

Reply:
If this was the only file that existed that would be the case, but because [the props file](https://github.com/jskeet/DemoCode/blob/c27cc15d83a372a3dd3346ee0df3fa3a350c55c6/DemoUtil/JonSkeet.DemoUtil/targets/JonSkeet.DemoUtil.props#L4) defines it and sets it to true, it's still true by default for an otherwise uninteresting project. The props file is automatically imported at the start of the project, and the targets at the end, so a project would have to explicitly opt out to turn it off.

TL;DR The example steps in the readme still work 😃

Reply:
Ah - cool, that's fine then. Admittedly I'm still slightly nervous of deviating from the Dragonfruit sample (as when they make a change I'll need to work out how it affects this) but I expect that will happen rarely.

Reply:
If this came from the Dragon fruit sample then I think I have another PR to open :)